File Manager/File Manager/Models/PasswordManager.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  PasswordManager.swift
+//  File Manager
+//
+//  Created by Ernest Mihasenko on 19.07.22.
+//
+
+import Foundation
+import KeychainSwift
+
+class PasswordManager {
+    
+    static let shared = PasswordManager()
+    private let keychain = KeychainSwift()
+    weak var delegate: PasswordManagerDelegate?
+    
+    private init() {}
+    
+    var hasPassword: Bool {
+        keychain.get("Password") != nil
```

Comment:
It's better to move `"Password"` into a separate property so you have the same key everywhere